spec/absolute.html

```diff
@@ -287,8 +287,7 @@ <h1>Temporal.Absolute.prototype.difference ( _other_ [, _options_ ] )</h1>
         1. Perform ? RequireInternalSlot(_other_, [[InitializedTemporalAbsolute]]).
         1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, « *"years"*, *"months"*, *"weeks"* », *"seconds"*).
         1. If ! CompareTemporalAbsolute(_absolute_, _other_) &lt; 0, then
-          1. Let _greater_ be _otherAbsolute_.
-          1. Let _smaller_ be _absolute_.
+          1. Throw a *RangeError* exception.
         1. Else,
```

Comment:
Nitpick, feel free to leave for another revision: It's now possible to remove the Else clause and `_greater_` and `_smaller_`. (Ditto in the other instances.)

Reply:
Good idea. Done. Resolving.